doc-surrealdb_versioned_docs/version-1.x/installation/running/docker.mdx

```diff
@@ -28,6 +28,8 @@ To get started using Docker, you can use the `latest` tag. To start a server use
 
 ```bash
 docker run --rm --pull always -p 8000:8000 surrealdb/surrealdb:latest start
+## Use the below command for local development
+# docker run --rm --pull always -p 8000:8000 surrealdb/surrealdb:latest-dev start
```

Comment:
I think it will be valuable to explain why using `latest-dev` for local development. It includes a shell (`sh`) and a package manager so user can install tooling and interact with the docker container internally

Reply:
I agree @sgirones . I'll add a note to this instead of commenting it out.

Reply:
Done @Ekwuno! Added a note with explanation instead of the previously commented out command. Added the update to V2 doc as well 👍🏽